tests/functional/intelligent-host-selection/00-mixedhost/flow.cylc

```diff
@@ -0,0 +1,23 @@
+
+[meta]
+title = "Try out scenarios for intelligent host selection."
+description = """
+Tasks
+- goodhost: a control to check that everything works
+- mixedhost contains some hosts that will and won't fail
+"""
```

Comment:
For small workflows like this, I think it's easier to put them inside the `.t` file using `init_workflow` or whatever it's called. Up to you though

Reply:
I think I agree. It makes debugging a 1-stop-shop. Will do. 👍🏼

Reply:
Actually, on second thoughts I have a good reason for _not_ doing this: I think it makes it easier to run the test workflow outside the test which is helpful for debugging.

Reply:
Fair enough